src/server/server.ts

```diff
@@ -535,6 +535,7 @@ namespace ts.server {
                 hrtime: process.hrtime,
                 logger,
                 canUseEvents,
+                suppressProjectEvents,

```

Comment:
Feels like this should be flags that indicates event to suppress given that this does not suppress ```ProjectInfoTelemetryEvent``` which is project event

Reply:
In the interest of keeping the name intelligible, I decided not to incorporate that.  They also don't seem conceptually related to projects.

Reply:
then may be rename it to onlyTelemetryEvent or something as such?

Reply:
It's not only telemetry.  We also want TypingsInstaller events.

Reply:
We could call it `suppressDiagnosticEvents`, but then I'd feel obliged to let `ProjectsUpdatedInBackgroundEvent` through (which is fine, it just requires a different implementation).

Reply:
Changed to `suppressDiagnosticEvents`.